SpravaOsobnichFinanci/SpravaOsobnichFinanci/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace SpravaOsobnichFinanci
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Sprava osobnich financi");
+
+            Console.WriteLine("\nZadej sve jmeno: "); // Jmeno
+            string odpoved = Console.ReadLine();
+
+            Console.WriteLine("Jmeno:" + odpoved); // Jmeno
+
+            Console.WriteLine("\nZadej svuj vek: "); // Vek
+            string odpoved1 = Console.ReadLine();
```

Comment:
Zde by bylo dobre vstup po nacteni prekonvertovat do spravneho datoveho typu. V pripade veku je to integer

Napr. takto:
```suggestion
            string vekOdpoved = Console.ReadLine();
            int vekUzivatele = int.Parse(vekOdpoved);
```